spec/predicates/cyclonedx.md

```diff
@@ -0,0 +1,71 @@
+# Predicate type: CycloneDX
+
+Type URI: (tentative) https://cyclonedx.org/bom
+
+Version: 1.0.0
```

Comment:
If this predicate is meant to comply with the latest CycloneDX spec version, this should probably be 1.4